third_party/move/move-compiler/tests/move_check/inlining/inline_accessing_constant.exp

```diff
@@ -3,4 +3,48 @@ bug[ICE12001]: BYTECODE GENERATION FAILED
   │
 9 │ module 0xc0ffee::dummy2 {
   │                  ^^^^^^ IR ERROR: Missing constant definition for CC
+Backtrace: Backtrace [
```

Comment:
Debugging output.  Better get rid of this to pass the tests.

Reply:
Done